src/components/badges/Badge.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { FlexWrapper, Image, Typography } from "components";
+
+interface Styles {
+    backgroundColor?: string;
+    width?: string;
+    height?: string;
+    marginLeft?: string;
```

Comment:
why don't you pass simply margin? 
you can pass all of these values in one line margin: 1rem 1rem 1rem 1rem; top, right, bottom, left